package.json

```diff
@@ -34,60 +34,55 @@
   "devDependencies": {
     "body-parser": "^1.17.1",
     "broccoli-asset-rev": "^2.4.5",
-    "broccoli-merge-trees": "^2.0.0",
```

Comment:
nope, it's usually used to adjust the build pipeline, but I'm not aware that Ember CLI ever added that as a project dependency